releasenotes/notes/snmp_add_tags_to_interface_configs-50283df5d3adabf7.yaml

```
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    [corechecks/snmp] Add ``tags`` to ``interface_configs`` to tag interface metrics
```

Non-blocking, do double back-ticks change the formatting compared to the single ` ?
```suggestion
    [corechecks/snmp] Add `tags` to `interface_configs` to tag interface metrics.
```